src/components/InvalidAssignment/InvalidAssignment.scss

```diff
@@ -0,0 +1,41 @@
+@import "../../pages/Index/index.scss";
+
+.InvalidAssignment {
+  background-color: #b1b0ff;
+  width: 100vw;
+  height: 7vw;
+  border-radius: 1em;
+  border: 2px solid black;
+  position: fixed;
+  top: 0;
+  z-index: 1;
+  margin: auto;
+}
+
+.Message {
+  padding-top: 1vw;
+  width: 90vw;
+  height: 4vw;
+  font-family: "Roboto", sans-serif;
```

Comment:
since you happened to use this font family 2x in the same file you should put it as a variable!! :)))